_data/chains/eip155-1111.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "WEMIX3.0 Mainnet",
+  "chain": "WEMIX",
+  "rpc": ["https://api.wemix.com", "wss://ws.wemix.com"],
+  "faucets": [],
+  "nativeCurrency": {
+    "name": "WEMIX",
+    "symbol": "WEMIX",
+    "decimals": 18
+  },
+  "infoURL": "https://wemix.com",
+  "shortName": "wemix",
+  "chainId": 1111,
+  "networkId": 1111,
+  "explorers": [
+    {
+      "name": "WEMIX Block Explorer",
+      "url": "https://explorer.wemix.com",
```

Comment:
this site seems to be down
![image](https://user-images.githubusercontent.com/111600/196557972-4b992324-20b2-4aea-9b8e-7b9635134ac7.png)

Reply:
Yes, it will be ready Oct 20th 3 AM UTC. Thanks.

Reply:
Please ping me then or remove the explorer for now - currently I cannot verify EIP3091 compatibilty and so can't merge

Reply:
Sure, I will ping you when it is up and ready to check. Thanks.

Reply:
Explorer is ready to use. Please confirm this.

Reply:
@ligi 

> Please ping me then or remove the explorer for now - currently I cannot verify EIP3091 compatibilty and so can't merge

could you check this? Wemix explorer is ready to use (https://explorer.wemix.com)

Reply:
@ligi could you please confirm?